sky/cli.py

```diff
@@ -5911,11 +5911,12 @@ def api_info():
     user_name = os.getenv(constants.USER_ENV_VAR, getpass.getuser())
     user_hash = common_utils.get_user_hash()
     dashboard_url = server_common.get_dashboard_url(url)
-    click.echo(f'Using SkyPilot API server: {url} Dashboard: {dashboard_url}\n'
+    click.echo(f'Using SkyPilot API server: {url}\n'
```

Comment:
Hmm, if we make the change here, do we want to keep the `sky api start` align with the output here, i.e. the output when API server is firstly started?

Reply:
Good catch, looks like this now:
```
 » sst
Failed to connect to SkyPilot API server at http://127.0.0.1:46580. Starting a local server.
✓ SkyPilot API server started.
├── Dashboard may be stale when installed from source, to rebuild: npm --prefix sky/dashboard install && npm --prefix sky/dashboard run build
└── Dashboard: http://127.0.0.1:46580/dashboard
```